src/api/users.js

```diff
@@ -0,0 +1,5 @@
+import { getData } from './handleData';
+
+export function getAllUsers(id = '') {
+  return getData(`/users/${id}`);
+}
```

Comment:
You don't use this function with id anywhere.

```suggestion
export function getAllUsers() {
  return getData(`/users`);
}
```